codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/alltypesresponse.java

```diff
@@ -1581,38 +1575,41 @@ public final void setStructWithNestedBlob(StructWithNestedBlobType.BuilderImpl s
         }
 
         public final Map<String, ByteBuffer> getBlobMap() {
-            return blobMap;
+            return blobMap == null ? null : blobMap.entrySet().stream()
+                                                   .collect(Collectors.toMap(e -> e.getKey(), e -> e.getValue().asByteBuffer()));
         }
 
         @Override
-        public final Builder blobMap(Map<String, ByteBuffer> blobMap) {
+        public final Builder blobMap(Map<String, SdkBytes> blobMap) {
             this.blobMap = BlobMapTypeCopier.copy(blobMap);
             return this;
         }
 
         public final void setBlobMap(Map<String, ByteBuffer> blobMap) {
-            this.blobMap = BlobMapTypeCopier.copy(blobMap);
+            blobMap(blobMap == null ? null : blobMap.entrySet().stream()
+                                                    .collect(Collectors.toMap(e -> e.getKey(), e -> SdkBytes.fromByteBuffer(e.getValue()))));
         }
 
-        public final Collection<ByteBuffer> getListOfBlobs() {
-            return listOfBlobs;
+        public final List<ByteBuffer> getListOfBlobs() {
+            return listOfBlobs == null ? null : listOfBlobs.stream().map(SdkBytes::asByteBuffer).collect(Collectors.toList());
```

Comment:
should we return empty list? same as map

Reply:
We're just trying to match the behavior of listOfBlobs() here.